test/policies/list_response_policy_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class ListResponsePolicyTest < ActiveSupport::TestCase
+
+  def test_scope
+  end
+
+  def test_show
+  end
+
+  def test_create
+  end
+
+  def test_update
+  end
+
+  def test_destroy
+  end
```

Comment:
Add tests or remove file

Reply:
actually this is all WIP, so that may explain some of the unfinished stuff.
But still highly appreciate your review 👍 👍